console/module/authorization/src/main/java/org/eclipse/kapua/app/console/module/authorization/client/tabs/permission/PermissionAddDialog.java

```diff
@@ -200,7 +200,10 @@ public void onSuccess(List<GwtAction> result) {
                         actionsCombo.setSimpleValue(allAction);
                         actionsCombo.enable();
 
-                        if (selectedDomain.getGroupable()) {
+                        if (selectedDomain.getGroupable() == null) {
+                            groupsCombo.setEnabled(true);
```

Comment:
From the DB it will never be `null`. For better understanding I would check this agains class variable `allDomain`  doing something like `allDomain.equals(selectedDomain)`